edge-modules/TestAnalyzer/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG base_tag=1.0.4-linux-arm64v8
```

Comment:
update base tag?

Reply:
Nope. We don't publish ARM64 since it's using Dotnet3.0.